app/helpers/application_helper.rb

```diff
@@ -373,23 +373,27 @@ def update_otml_url_for(component, escape=true)
     end
   end
 
-  def report_link_for(reportable, action='report', link_text='Report ', title=nil)
-    return "" if reportable.respond_to?('reportable?') && !reportable.reportable?
+  def learner_report_link_for(learner, action='report', link_text='Report ', title=nil)
+    return "" if learner.respond_to?('reportable?') && !learner.reportable?
 
-    reportable_display_name = reportable.class.display_name.downcase
+    reportable_display_name = learner.class.display_name.downcase
     action_string = action.gsub('_', ' ')
-    name = reportable.name
+    name = learner.name
     format = nil
-    if (reportable.respond_to?('runnable') && reportable.runnable.respond_to?('report_format') && reportable.runnable.respond_to?('report_url') && reportable.runnable.report_url && !reportable.runnable.report_url.strip.empty?)
-      format = reportable.runnable.report_format
-    end
-    url = polymorphic_url(reportable, :action => action, :format => format)
+
+    url = polymorphic_url(learner, :action => action, :format => format)
```

Comment:
format looks like it is always nil here so I think it could be removed entirely.